harness/determined/common/api/bindings.py

```diff
@@ -3069,6 +3051,44 @@ def to_json(self) -> typing.Any:
             "limit": self.limit if self.limit is not None else None,
         }
 
+class v1PatchExperiment:
+    def __init__(
+        self,
+        id: int,
+        description: "typing.Optional[str]" = None,
+        forkedFrom: "typing.Optional[int]" = None,
+        labels: "typing.Optional[typing.Sequence[typing.Dict[str, typing.Any]]]" = None,
+        name: "typing.Optional[str]" = None,
+        notes: "typing.Optional[str]" = None,
+    ):
+        self.id = id
+        self.description = description
+        self.labels = labels
+        self.name = name
+        self.notes = notes
+        self.forkedFrom = forkedFrom
+
+    @classmethod
+    def from_json(cls, obj: Json) -> "v1PatchExperiment":
+        return cls(
+            id=obj["id"],
+            description=obj["description"] if obj.get("description", None) is not None else None,
+            labels=obj["labels"] if obj.get("labels", None) is not None else None,
+            name=obj["name"] if obj.get("name", None) is not None else None,
+            notes=obj["notes"] if obj.get("notes", None) is not None else None,
```

Comment:
unrelated couldn't `obj["notes"] if obj.get("notes", None) is not None else None,` be just `obj.get("notes", None)` did I mess this up in the genrator?

Reply:
`None) is not None else None,` appears in a bunch of places in the bindings file, agree it would be cleaner if it didn't generate this way